### PR TITLE
yugabyte docs: add directions on how to access yugabytedb dashboard

### PR DESCRIPTION
**Pull request by hasheddan**
**Description of your changes:**

Updates the YugabyteDB docs to show a user how to access the Yugabyte dashboard locally. Just a nice feature when going through the quickstart!

**Which issue is resolved by this Pull Request:**
Resolves N/A

[skip ci]

**Checklist:**

- [x] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [x] Documentation has been updated, if necessary.
- [x] Unit tests have been added, if necessary.
- [x] Integration tests have been added, if necessary.
- [x] Pending release notes updated with breaking and/or notable changes, if necessary.
- [x] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [x] Code generation (`make codegen`) has been run to update object specifications, if necessary.
- [x] Comments have been added or updated based on the standards set in [CONTRIBUTING.md](https://github.com/rook/rook/blob/master/CONTRIBUTING.md#comments)
- [x] Add the flag for skipping the CI if this PR does not require a build. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for more details.

### CODE REVIEW

**Comment by travisn** [comment]
@hasheddan Please force push to the PR to trigger the CI. There was a CI issue yesterday preventing the builds from starting.

**Comment by hasheddan** [comment]
@travisn happy to do so. However, I included [skip ci] in the PR body per this [doc](https://github.com/rook/rook/blob/90e24296c428b7c0040a0e205e1ffb6b2c1e58cc/INSTALL.md#skip-ci). Just want to confirm that it should be removed?

**Comment by travisn** [comment]
You should keep the `[skip ci]` since it's a doc change. We just still need to trigger the CI so it can turn the CI check green. Yeah, kind of strange that you have to start the CI in order to skip it. :)

**Comment by hasheddan** [comment]
@travisn gotcha, thanks for your patience :) rebased and pushed!

File: Documentation/yugabytedb.md [hunk 1]
```diff
@@ -87,6 +87,14 @@ kubectl -n rook-yugabytedb exec -it yb-tserver-hello-ybdb-cluster-0 /home/yugaby
 ```
 Refer the [YugabyteDB documentation](https://docs.yugabyte.com/latest/quick-start/explore-ysql/#kubernetes) for more details on the `ysql` api.
 
+You can also access the YugabyteDB dashboard using port-forwading.
```
Comment (travisn) [inline Documentation/yugabytedb.md]: typo: port-forwarding

Comment (hasheddan) [reply Documentation/yugabytedb.md]: No worries at all! Will fix this now

**Pull request merged**